### ISSUE TITLE
Default character and ability to deactivate ElevenLabs voice

### ISSUE THREAD

**Issue by stevenbaert** [Shaunwei/RealChar#316]
Love this project, which is up and running on my Windows 11 machine now.
Though 2 feature requests, which would be huge improvement:

<referring again to other project https://github.com/JavaFXpert/talk-with-gpt3>
- if default page load is character last used or a default chosen, then you can just run it always on the background and wake it up , put it to sleep = personal assitant
- Voice feature is great, but expensive if you want to use the assistant intensively <see screenshot below>, the talk-with-gpt3 uses Polly (Amazon) which looks (didn't get into details yet) much cheaper(?)
-might be intresting to be able to disable voice alltogether, also to limit costs (and that voice preference would be saved)
-using RealChar as alternative to ChatGPT is a perfect valid alternative in professional environment (= personal professional assistant), though the avatars on the screen, makes it looks like as if you are playing around/playing games, would be nice to be able to tune layout in config-file, like having the ChatGPT UI (no avatars)

![image](https://github.com/Shaunwei/RealChar/assets/43472567/89d710db-bc25-490e-b304-86c20bd61eb3)

**Comment by pycui** [comment]
Thanks Steven! This is really good feedback. For any of below, if you want to contribute, feel free to try it and send us a PR or just let us know your ideas. 

1. For the wake-up: we are working on it. For now, a less-ideal workaround is you can mute the mic when you are not using it (e.g. phone hang-up state), and unmute when you want to say something. 
2. Amazon Polly sounds interesting - I haven't tried it out, but if you have done it (and it works), feel free to add a new TTS component in `realtime_ai_character/audio/text_to_speech/__init__.py`, it should be pretty easy. We also have some other TTS options than ElevenLabs that's cheaper (google TTS, unreal speech)
3. Disabling voice - we can look into making this an option - but hopefully you can still enjoy voice with cheaper options, because Voice is the magic here for good experience. 
4. Professional look: this is currently not the direct focus - we don't want to just make a chatGPT UI. But if you have made some UI already, we can incorporate it too!

**Comment by stevenbaert** [comment]
Thanks for your feedback. Regarding the GUI, please seey other post on https://github.com/dotneet/smart-chatbot-ui

Would be my ideal scenario to be able to use RealChar as a full replacement of my (daily) ChatGPT interactions plus having customizability like choosing Azure OpenAI-Service and adding docs/memory!

Though the way RealChar works now, it is not.

Happy to go more into details or even make boilerplate design of how I see that.
Basically it would be like the Smart-chatbot-ui but then within options you could choose for enabling/disabling features. In a top menu you would have quick choices like mute/disable output/voice, button for sleep (wake/sleep word would be "wake up" and "go to sleep").
That way you could even talk to RealChar when window not open/in background.
